parkhangs-webapp/src/features/modal/ModalParkDetail.js

```diff
@@ -118,7 +118,7 @@ class ModalParkDetail extends React.Component {
                             </div>
                         </div>
                     </div>
-                    <div className="Section EventTable">
+                    <div className="Section" id = "EventTable">
```

Comment:
my linter throws errors because of this space haha

Reply:
okay, lemme take this out; was playing with styling